Skatepark: query loop block pattern

MaggieCabrera opened the pull request:
<!-- Thanks for contributing to our free themes! Please provide as much information as possible with your Pull Request by filling out the following - this helps make reviewing much quicker! -->

#### Changes proposed in this Pull Request:

As discussed on slack, this PR brings in the index.html query loop as a pattern since we didn't have any pattern for the block:

<img width="1373" alt="Screenshot 2021-08-19 at 17 39 33" src="https://user-images.githubusercontent.com/3593343/130099117-3d14c51e-ab85-4f48-84d3-8c8d84e1c4b7.png">

MaggieCabrera committed:
query loop block pattern

kjellr commented on skatepark/inc/patterns/blog-posts.php:
```diff
@@ -0,0 +1,36 @@
+	<hr class="wp-block-separator is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:post-featured-image {"style":{"color":{"duotone":["#000","#B9FB9C"]}}} /-->
```
We should make sure `isLink` is true for the featured images here (same for the actual template itself)

melchoyce commented:
What's up with the link placeholder alignment here? 👀 

![image](https://user-images.githubusercontent.com/2846578/130110175-ecace99a-f03d-4c05-894f-74d440b065df.png)

MaggieCabrera committed:
added link to featured image, fixed indent on read more link

MaggieCabrera commented:
Added image link to the pattern and the template and fixed the read more text (the text-indent from the except was being applied to the paragraph that contains the read more link)

mikachan reviewed (commented):
This is looking good on my side! The only things I can spot compared to the design is an underline on the dates, or maybe these should be links?

And should the 'read more' link have a heavier font-weight, maybe `500`?

Here's the design I'm looking at:

![Screenshot 2021-08-20 at 12 24 14](https://user-images.githubusercontent.com/1645628/130226409-e3bbfe64-9074-4f4a-b790-0eb7c831adac.png)

kjellr commented:
Yeah, I'll second the notes from @mikachan. Also, the vertical spacing between all the post meta (featured image, title, excerpt, etc.) is all much greater than it is in the comps. Can we make that a little closer to the mockups? 

Comps|This PR
---|---
<img width="202" alt="Screen Shot 2021-08-20 at 8 54 40 AM" src="https://user-images.githubusercontent.com/1202812/130236546-a92cc709-5fe0-4fc5-9ed3-0a467cfef8f1.png">|<img width="202" alt="Screen Shot 2021-08-20 at 8 54 54 AM" src="https://user-images.githubusercontent.com/1202812/130236573-2a4ad441-e310-4959-9174-29fdc62bf34f.png">

jffng commented:
> Also, the vertical spacing between all the post meta (featured image, title, excerpt, etc.) is all much greater than it is in the comps. Can we make that a little closer to the mockups?

I've noticed this too during most of my attempts to make a pattern in Skatepark, and I think it has to do with this: https://github.com/Automattic/themes/blob/trunk/skatepark/child-theme.json#L203

Should we reduce the value of this variable to something closer to the vertical spacing between the text in the comps?

MaggieCabrera committed:
fixed spacing issues on the query block

MaggieCabrera commented:
I reduced the vertical spacing from 45 to 40px, I think it's more fitting to the design. The biggest problem about the spacing is that we were applying the same distance between the headings and the rest of the content and between every paragraph and the rest. According to figma, the distance between paragraphs should be 20px instead so I override that for Skatepark on the CSS and that makes the theme look like the design. 

I've created a [separate issue to support RTL](https://github.com/Automattic/themes/issues/4453) with the arrow so that we don't block this PR with that issue.

scruffian commented on skatepark/inc/patterns/blog-posts.php:
```diff
@@ -0,0 +1,36 @@
+	
+	<!-- wp:post-title {"isLink":true,"fontSize":"normal"} /-->
+	
+	<!-- wp:post-excerpt {"moreText":"Continue reading","className":"is-style-indented-post-excerpt"} /-->
```
This text should be translated.

MaggieCabrera replied:
Great catch thank you!

MaggieCabrera committed:
translatable read more text

scruffian commented on skatepark/block-templates/index.html:
```diff
@@ -5,12 +5,12 @@
+	<!-- wp:post-featured-image {"isLink":true,"style":{"color":{"duotone":["#000","#B9FB9C"]}}} /-->
 	
 	<!-- wp:post-title {"isLink":true,"fontSize":"normal"} /-->
 	
```
Should we add the same more text to this template?

MaggieCabrera replied:
it's not translatable there...

scruffian replied:
I know, but without it there's no link at all. I guess that's still better than a link that's not translated.

scruffian reviewed (approved):
LGTM
